### DISCUSSION TITLE
Red Eclipse in the whisker menu

### DISCUSSION THREAD

**Discussion by fishyfngers** [redeclipse/base#1419] [Help and Support] [answered]
I downloaded Red Eclipse on Arcolinux xfce and I don't know how to get Red Eclipse into the whisker menu at all.  The application finder can't see it either.  I can start the game from the base folder but I was wondering if there was a way to get the game to open from the whisker menu or application finder.  I have the base folder where the game itself is and there is a hidden  ".redeclipse" folder that seems to not have much in it.  I am fairly new to Linux so pardon me if it sounds like a dumb question.

**Comment by qreeves** [comment, marked as answer]
Install and use [MenuLibre](https://bluesabre.org/projects/menulibre) to create entries in your applications menus on Linux.

**Reply by fishyfngers** [reply]
Thank you.  That did the trick.

**Reply by qreeves** [reply]
No problem @fishyfngers 😉
